client/src/components/Form.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class Form extends React.Component {
+  render() {
```

Comment:
It looks like this isn't used, are you planning to use it in the future? If not, it should probably be deleted.